SECURITY.md

```diff
@@ -5,22 +5,23 @@ If you have any concerns please let us know.
 
 ## Supported Versions
 
-The `main`, and `1.7.x`, branches are fully supported by the upstream authors.
-Additionally, the `1.5.x` and `1.6.x` branches are supported for security fixes.
+The `main`, and `1.8.x`, branches are fully supported by the upstream authors
+with all unstable code belonging in `wip/{USERANME}/{TOPIC}` branches.
```

Comment:
Username spelled wrong and I think it's better to just say topic branches.

Reply:
How about this?